source/form.html

```diff
@@ -0,0 +1,169 @@
+                  <input class="" type="email" name="mail" id="mail" placeholder="Введите e-mail">
+                </div>
+              </div>
+            </fieldset>
+            <fieldset class="revocation-form__item">
+              <div class="revocation-form__wrapper-item">
+                <legend class="revocation-form__title">Ваше Общее впечатление:</legend>
+                <div class="revocation-form__your-impression">
+                  <input class="" type="radio" name="positively" id="positively" value="" checked>
+                  <label for="positively">Скорее положительное</label>
+                </div>
+                <div class="revocation-form__your-impression">
+                  <input class="" type="radio" name="negative" id="negative" value="">
+                  <label for="negative">Скорее отрицательное</label>
+                </div>
+                <div class="revocation-form__your-impression">
+                  <input class="" type="radio" name="neutrality" id="neutrality" value="">
+                  <label for="neutrality">Затрудняюсь ответить</label>
+                </div>
+              </div>
+            </fieldset>
+            <fieldset class="revocation-form__item">
+              <div class="revocation-form__wrapper-item">
+                <legend class="revocation-form__title">посещенные достопримечательности:</legend>
+                <div class="revocation-form__sights-visited">
+                  <input type="checkbox" name="devil's-bridge" id="devil's-bridge" value="" checked>
+                  <label for="devil's-bridge">Мост Дьявола</label>
+                </div>
+                <div class="revocation-form__sights-visited">
+                  <input type="checkbox" name="mount-bell" id="mount-bell" value="" checked>
+                  <label for="mount-bell">Гора-Колокол</label>
+                </div>
+                <div class="revocation-form__sights-visited">
+                  <input type="checkbox" name="slide-park" id="slide-park" value="" checked>
+                  <label for="slide-park">Слайд-парк</label>
+                </div>
+                <div class="revocation-form__sights-visited">
+                  <input type="checkbox" name="red-rocks" id="red-rocks" value="">
+                  <label for="red-rocks">Красные скалы</label>
+                </div>
+              </div>
+            </fieldset>
+            <fieldset class="revocation-form__item">
+              <div class="revocation-form__wrapper-item">
+                <legend class="revocation-form__title">Опишите свои эмоции:</legend>
+                <div class="revocation-form__emotion">
+                  <label for="emotion"></label>
+                  <textarea name="emotion" id="emotion" rows="8" cols="80"></textarea>
+                </div>
+              </div>
+              <div class="revocation-form__post-a-review">
+                <input type="submit" name="button" value="отправить отзыв">
```

Comment:
А почему не `button`?

```suggestion
                <button type="submit" name="button">отправить отзыв</button>
```